packages/backend/server/src/core/doc/manager.ts

```diff
@@ -404,6 +410,54 @@ export class DocManager implements OnModuleInit, OnModuleDestroy {
     return null;
   }
 
+  async getPageContent(
+    workspaceId: string,
+    guid: string
+  ): Promise<PageDocContent | null> {
+    const cacheKey = `workspace:${workspaceId}:doc:${guid}:content`;
+    const cachedResult = await this.cache.get<PageDocContent>(cacheKey);
+
+    if (cachedResult) {
+      return cachedResult;
+    }
+
+    const doc = await this.get(workspaceId, guid);
+    if (!doc) {
+      return null;
+    }
+
+    const content = parsePageDoc(doc.doc);
+
+    if (content) {
+      await this.cache.set(cacheKey, content, { ttl: 60 * 1000 });
```

Comment:
only 1 min? i think this can be longer, we can prevent the cache if receive the doc update